cdap-integration-test/src/main/java/co/cask/cdap/test/IntegrationTestBase.java

```diff
@@ -366,16 +366,22 @@ protected DatasetClient getDatasetClient() {
     return new DatasetClient(getClientConfig(), getRestClient());
   }
 
-  protected Id.Namespace createNamespace(String name) throws Exception {
-    Id.Namespace namespace = new Id.Namespace(name);
+  protected NamespaceId createNamespace(String name) throws Exception {
```

Comment:
``IntegrationTestBase`` is not public API right? If its not public API, maybe it should not be in the CDAP codebase, but that question is for a later date.